### ISSUE TITLE
Delete/Adding issue with arrays on a tabs layout 

### ISSUE THREAD

**Issue by michaelviegas** [jdorn/json-editor#444]
Hi,

First of all, congrats for this amazing library. Is really good stuff!

Recently I came across with an issue when I delete and add an item after on a array with a "tabs" layout. This issue doesn't happen if I use a "table" layout. Here's a link: http://jeremydorn.com/json-editor/?schema=N4IgLgngDgpiBcICGAnFSIgDQgGYHsUBbJMBcJAIwGdsQBLMGI2+USWc/SgKxgGMyOABYwkAExgoAKsygAbUnETBg1GPNwA6AHZIiMAL6G6UFPlgow9GK1B6DCdtGUhqYFPR0BzOtbDyrgBy+nDGxkAA&value=NobwRAdghgtgpmAXGAZgSwE4GcAuACNHOGPMAXwBpxp4kws4BjAewgBMCiZyrJYFkOABaYOhYuQC6QAA&theme=bootstrap2&iconlib=fontawesome4&object_layout=normal&show_errors=interaction

Follow this steps to reproduce the issue:
1. Delete the first item
2. Add a new one
3. The last item gets duplicated and added to the array.

Another related issue:
1. Delete all items
2. Have fun by watching zombie items come back when adding one by one :)

Cheers
Michael

**Comment by stevage** [comment]
The "related issue" is probably #292, which I have submitted a pull request for.

**Comment by KrishnaPG** [comment]
Yes. Occasionally, the fields (edit boxes) are also disappearing (screenshot below):

![image](https://cloud.githubusercontent.com/assets/4610831/8780057/48739100-2f24-11e5-9dac-b9519747370d.png)
